__tests__/index.test.ts

```diff
@@ -28,15 +31,35 @@ jest.mock('@actions/github', () => {
     }
   }
 })
+jest.mock('@actions/github/lib/utils')
 jest.mock('@electron/github-app-auth')
 
 jest
   .mocked(appCredentialsFromString)
   .mockReturnValue({ appId: '12345', privateKey: 'private' })
 
+const getAuthenticated = jest.fn()
+const getByUsername = jest.fn()
+
+;(GitHub as unknown as jest.Mock).mockReturnValue({
```

Comment:
I beg we just use semicolons properly?

Reply:
Yea, it hurt a bit to add that, but the linter insists. 😅 This repo is created from the `actions/typescript-action` template which is configured to not use semicolons. I can look at a follow-up PR to change that.